### PR TITLE
May month minutes of meeting

### PR DESCRIPTION
**Pull request by gnurenga**

### CODE REVIEW

File: content/may-2016-meet-minutes.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+Title: May Meet Minutes
+Date: 2016-06-10 11:49
+Tags: Meeting Minutes
+
+
+After a long time we were back to IMSc Ramanujam auditorium.  The
+meetup Intro was given by Abhishek.
+
+We started with second talk since Shreyas was not available for this
```
Comment (bravegnu) [inline content/may-2016-meet-minutes.md]: Shreyas -> Shrayas

File: content/may-2016-meet-minutes.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+meetup.
+
+## Introduction to Celery
+
+The first talk was given by Shanmuga. He started explaining how and
+why he started using celery in his project. He continued with
```
Comment (bravegnu) [inline content/may-2016-meet-minutes.md]: celery -> Celery

File: content/may-2016-meet-minutes.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+requirement for setting up celery like choosing a broker (RabbitMq or
```
Comment (bravegnu) [inline content/may-2016-meet-minutes.md]: celery -> Celery

File: content/may-2016-meet-minutes.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+Redis etc). He explained the advantages and disadvantages of those
+brokers. Then he started explaining a prototype written by him.  I am
+not available after some 4 slides since myself and vengatesh left to
```
Comment (bravegnu) [inline content/may-2016-meet-minutes.md]: I *was* not available

Comment (bravegnu) [inline content/may-2016-meet-minutes.md]: vengatesh -> Vengatesh

File: content/may-2016-meet-minutes.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+alert tea for networking tea.
+
+## Short Introduction by every one.
+
+When I came back from canteen audience gave intro to themselves.
+Mostly every one explain why they are coming for the meetup.
```
Comment (bravegnu) [inline content/may-2016-meet-minutes.md]: everyone *explained*

File: content/may-2016-meet-minutes.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+Since the tea is little bit late we continued with next session.
+
+## Code Optimization in Python
+
+The second talk is given by Ashok on Code Optimization.  He started
```
Comment (bravegnu) [inline content/may-2016-meet-minutes.md]: second talk *was* given

File: content/may-2016-meet-minutes.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+the session by asking questions related to Optimization like Why
+optimization is require ? where we need to apply this concept ?  What
```
Comment (bravegnu) [inline content/may-2016-meet-minutes.md]: where -> Where

Comment (bravegnu) [inline content/may-2016-meet-minutes.md]: is *required*

File: content/may-2016-meet-minutes.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+are all the things we need take care in Optimization ?  He explained
+profiling in python and gave some reference.
```
Comment (bravegnu) [inline content/may-2016-meet-minutes.md]: python -> Python

File: content/may-2016-meet-minutes.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+
+## Networking Tea Break
+
+## Deep Learning with Tensor Flow
+
+The third talk was about Tensor Flow by Manish.  He explained how
+tensor flow can use GPU to compute expressions in parallel.  If my
```
Comment (bravegnu) [inline content/may-2016-meet-minutes.md]: Tensor Flow

File: content/may-2016-meet-minutes.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+understanding is right, if there is an equation or problem which
+relies input from other 10 equation in order to solve then the tensor
+flow will evaluate each expression or problem parallel and give 10
```
Comment (bravegnu) [inline content/may-2016-meet-minutes.md]: or problem *in* parallel

File: content/may-2016-meet-minutes.md [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+inputs to solve the final equation.  He actually porting
+his hand written nural application code to tensor flow.
```
Comment (bravegnu) [inline content/may-2016-meet-minutes.md]: neural ?

**Comment by bravegnu** [comment]
Thanks Rengaraj for the minutes. There were a few issues that I have pointed out in the line comments.

File: content/may-2016-meet-minutes.md [hunk 12]
```diff
@@ -6,30 +6,30 @@ Tags: Meeting Minutes
...
-why he started using celery in his project. He continued with
-requirement for setting up celery like choosing a broker (RabbitMq or
+why he started using Celery in his project. He continued with
+requirement for setting up Celery like choosing a broker (RabbitMq or
 Redis etc). He explained the advantages and disadvantages of those
-brokers. Then he started explaining a prototype written by him.  I am
-not available after some 4 slides since myself and vengatesh left to
+brokers. Then he started explaining a prototype written by him.  I was
+not available after some 4 slides since myself and Vengatesh left to
 alert tea for networking tea.
```
Comment (kirang89) [inline content/may-2016-meet-minutes.md]: "alert tea for networking tea." could be better phrased as "check the status of snacks/tea for the networking break ahead."

**Pull request merged**